src/apps/search-result/facet-browser.test.ts

```diff
@@ -0,0 +1,247 @@
+const coverUrlPattern = /^https:\/\/res\.cloudinary\.com\/.*\.(jpg|jpeg|png)$/;
+
+describe("Facet Browser", () => {
+  // I have used it.only because it is best practice to have self contained tests
+  // https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests
+  it.only("Open facet browser and check if all the facets are there", () => {
+    cy.contains(".disclosure__text", "Main languages");
+    cy.contains(".disclosure__text", "Access types");
+    cy.contains(".disclosure__text", "Children or adults");
+    cy.contains(".disclosure__text", "Creators");
+    cy.contains(".disclosure__text", "Fiction or nonfiction");
+    cy.contains(".disclosure__text", "Genre and form");
+    cy.contains(".disclosure__text", "Material types");
+    cy.contains(".disclosure__text", "Subjects");
+    cy.contains(".disclosure__text", "Work types");
+  });
+
+  it.only("Check if Main languages has all its terms", () => {
+    cy.contains(".disclosure__text", "Main languages").click();
+    cy.contains(".facet-browser__facet-group button", "Engelsk");
+    cy.contains(".facet-browser__facet-group button", "flere sprog");
+    cy.contains(".facet-browser__facet-group button", "Tysk");
+    cy.contains(".facet-browser__facet-group button", "Svensk");
+    cy.contains(".facet-browser__facet-group button", "Dansk");
+    cy.contains(
+      ".facet-browser__facet-group button",
+      "Sproget kan ikke bestemmes"
+    );
+    cy.contains(".facet-browser__facet-group button", "Fransk");
+    cy.contains(".facet-browser__facet-group button", "Latin");
+    cy.contains(".disclosure__text", "Main languages").click();
+    cy.get(`[aria-label="Close facet browser modal"]`).click();
+  });
```

Comment:
Please consider reducing this.

I see a lot of work went into writing all this but I do not think we have any reason to believe that one facet would behave differently than another. I would consider it sufficient to test that all facets are rendered and that all terms within one facet are rendered.